src/test/scala/edu/berkeley/ce/rockslicing/BlockVTKSpec.scala

```diff
@@ -0,0 +1,139 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest._
+import scala.math.sqrt
+
+class BlockVTKSpec extends FunSuite {
+  val boundingFaces = List(
+    Face((-1.0, 0.0, 0.0), 0.0, phi=0, cohesion=0), // -x = 0
+    Face((1.0, 0.0, 0.0), 1.0, phi=0, cohesion=0),  // x = 1
+    Face((0.0, -1.0, 0.0), 0.0, phi=0, cohesion=0), // -y = 0
+    Face((0.0, 1.0, 0.0), 1.0, phi=0, cohesion=0),  // y = 1
+    Face((0.0, 0.0, -1.0), 0.0, phi=0, cohesion=0), // -z = 0
+    Face((0.0, 0.0, 1.0), 1.0, phi=0, cohesion=0)   // z = 1
+  )
+  val unitCube = Block((0.0, 0.0, 0.0), boundingFaces)
+
+  val face1 = Face((-1.0, 0.0, 0.0), 0.0, phi=0, cohesion=0)
+  val face2 = Face((1.0, 0.0, 0.0), 1.0, phi=0, cohesion=0)
+  val face3 = Face((0.0, -1.0, 0.0), 0.0, phi=0, cohesion=0)
+  val face4 = Face((0.0, 1.0, 0.0), 1.0, phi=0, cohesion=0)
+  val face5 = Face((0.0, 0.0, -1.0), 0.0, phi=0, cohesion=0)
+  val face6 = Face((0.0, 0.0, 1.0), 1.0, phi=0, cohesion=0)
+  val face1Verts = List((0.0, 1.0, 0.0), (0.0, 0.0, 0.0), (0.0, 0.0, 1.0), (0.0, 1.0, 1.0))
+  val face2Verts = List((1.0, 1.0, 1.0), (1.0, 0.0, 1.0), (1.0, 0.0, 0.0), (1.0, 1.0, 0.0))
+  val face3Verts = List((0.0, 0.0, 1.0), (0.0, 0.0, 0.0), (1.0, 0.0, 0.0), (1.0, 0.0, 1.0))
+  val face4Verts = List((0.0, 1.0, 0.0), (0.0, 1.0, 1.0), (1.0, 1.0, 1.0), (1.0, 1.0, 0.0))
+  val face5Verts = List((1.0, 1.0, 0.0), (1.0, 0.0, 0.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0))
+  val face6Verts = List((0.0, 1.0, 1.0), (0.0, 0.0, 1.0), (1.0, 0.0, 1.0), (1.0, 1.0, 1.0))
+
+  def doubleListElementDiff(list1: Seq[Double], list2: Seq[Double]): Seq[Double] = {
+    assert(list1.length == list2.length)
+    val listDiff  = Seq.empty[Double]
+    for (i <- 0 until list1.length) {
+      listDiff +: Seq[Double](math.abs(list1(i) - list2(i)))
+    }
+    listDiff
+  }
+
+  def intListElementDiff(list1: Seq[Int], list2: Seq[Int]): Seq[Int] = {
+    assert(list1.length == list2.length)
```

Comment:
See previous comment. If I was right about `zip`, then you can do something nice like
```scala
list1.zip(list2) map { case (a,b) => a - b }
```